sys/include/stdio_base.h

```diff
@@ -26,11 +26,64 @@
 #include <unistd.h>
 
 #include "modules.h"
+#include "isrpipe.h"
+#include "xfa.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#ifndef STDIO_RX_BUFSIZE
+/**
+ * @brief Buffer size for STDIO
+ */
+#define STDIO_RX_BUFSIZE    (64)
+#endif
+
+enum {
```

Comment:
Where are those actually used? The only occurrence I see is in the arguments to `STDIO_PROVIDER`, and there these values are just added to the symbol in the preprocessor (and never hit the C language).

Reply:
True, I added them for future use if the need to select a certain stdio method comes up.

But I think this was when I thought the high lever should be responsible for managing the stdio connection (in case of remote shells) before I found that this is better handled by the stdio backend.